feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenEvent.kt

```diff
@@ -15,6 +15,7 @@ sealed interface HomeScreenEvent {
         object Start : FeedingEvent
         object Cancel : FeedingEvent
         object Finish : FeedingEvent
+        data class Expired(val rejectMessage: String) : FeedingEvent
```

Comment:
Let's leave this event as an object and put the `rejectMessage` inside as a constant. 
I guess we don't need localisation for such case.